external/downstream/pasture/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-if(ENABLE_psi4pasture)
-  ExternalProject_Add(psi4pasture
-    DEPENDS psi4_core
+if(ENABLE_pasture)
+  ExternalProject_Add(pasture
+    DEPENDS psi4-core
     GIT_REPOSITORY https://github.com/psi4/psi4pasture
     CMAKE_ARGS -DENABLE_ccsort=${ENABLE_ccsort}
                -DENABLE_transqt2=${ENABLE_transqt2}
```

Comment:
Are you running against a 554-patched psi4? I killed off the `<INSTALL_PREFIX>/external` dir (collapsed to install alongside psi4 itself) so that that wouldn't happen. Or using a pre-built pybind11 by chance? Because the line you suggested isn't necessary valid if pybind11 is detected external.